### DISCUSSION TITLE
小狗历险记 | /story/dog-find-her/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#640] [Announcements]
夏天尚未走远，大地仍然翠绿，茂密的草地上疏疏落落地飘着几片枯叶，秋天依稀还在遥远的未来。经历漫长的离别后，阿木小狗终于等到了心爱的主人，世界

https://yuanfan.rbind.io/story/dog-find-her/

**Comment by HushWay** [comment]
看完啦，有种很，emm，奇妙的感觉。这篇小说很宏大，凡酱的脑洞可以说是不着边际，剧情的走向一直在突破我的想象。复杂的世界观下，又藏好多细节，我看了好几遍，还是晕乎乎的，你写的时候没少费脑细胞吧。而且结局全是开放的，看来凡酱也是管杀不管埋的主……

根据我的理解，阿木在两条明线与一条暗线之间穿梭，线一是高居士引发的对世俗的探讨，寺庙立在江湖之中，袈裟穿在俗人身上，有这首词的感觉：

> 袈裟本无清静，红尘不染性空，幽幽古刹千年钟，都是痴人说梦

线二是淳风与永清社会身份本不相容，但是英雄相惜，最后又因为利益破裂。风能力与水能力的使用挺有意思的，不但可以杀人越货，还可以居家旅行，不错不错，学到了。

线三是一个被锁链束缚的灵魂，应当是凡酱自身意识的投影，有几分和世界格格不入的痛苦……

剧情很棒了，但有些行为动机我还理解不了，高居士绝望了为什么要杀狗呢？他为什么会觉得自己有罪孽呢？淳风和永清的情谊是怎么建立起来的呢？淳风如此爱护一只狗，可为什么能置人类和平于不顾去取玄门灵丹呢？

最后，你真的好会取名字，淳风、永清，好听又有寓意，搞得我有点羞愧，凡凡酱。阿木的名字有两个来源，其一衍生自阿鸣的一鸣惊人，其二来自淳风的无形之风与有形之木。我很好奇现实生活中阿木的起源是什么呢？

**Reply by earfanfan** [reply]
唐代有个数学家名叫李淳风，他和袁天罡还有一个关于推背图的传说。以前我想一些小说名字的时候就想过一对双胞胎分别叫做永清和永澈，不过武汉的汉口江滩也有一道门的名字是永清门。其实单独看这两个名字好像也不怎么样，不过我自己觉得淳风和永清作为对照来写就多了点意思。

2019年5月我刚领养阿木狗的时候给一个朋友说起，她以前捡了一只猫取名叫曼妮（PS就是 money 的谐音），她说也要给我的小狗取一个招财进宝的名字，我说那总不能直接叫它人民币吧，于是她说可以叫说 RMB 的短前音 RM，就是阿木了。

第一个高居士的故事取材于我在网络上看到的一个帖子，原贴已经找不到了，大致是帖子主人说他有个亲戚出家但是被骗光家财后赶出了寺庙。当然啦，我写高居士这个人物的故事本身也写的很隐晦，在我原先的设想中，该是这样的，高居士本来是有钱人家，厌恶了世俗虚伪后想到去出家，本来凭他最开始给寺庙的捐款是可以选法号选房间的，后来二和尚哄骗了他的许多钱财，又把他调去睡通铺，也使唤他劳动，等到新的一年新的有钱人出家后，他就被赶走了，当了几个月乞丐后受不了生活巨大反差带来的打击，于是最后摸回去寺庙下药放火一条龙，他杀狗只是想用狗做诱饵引阿鸣回来灭口。当然啦，这么多细节我都没写，纯粹只是因为我现在根本没有能力把控好文字，稍微写长点就……就写不动了。

关于这个故事我确实也是有些思考的，我们这个社会被儒家统治了几千年，可是儒家的那些所谓“名正言顺”的礼教束缚，在我看来非常虚伪，很多人都会下意识的以为有“名”的多少有点“实”，然而儒教统治了多久，名与实的分离就诞生了同样之久。很多人以为出家就可以逃离世俗尘网，可是寺庙一样身在红尘，如何能远离得了呢。

关于淳风和永清，我写之前心里想的是，她俩个原本是最好的朋友，是同门，只是因为所选的心中之道不同所以选择了不同的社会身份，也走了不同的路，原本设定的是永清病了于是淳风去偷灵丹，在永清的眼里灵丹被偷世界就会陷入战乱，她认为这样做太自私了，可是淳风却觉得不管灵丹是否存在，世界本来一直就是分久必合合久必分的，所谓灵丹与世界和平的关系也不过只是一层束缚永清思想的“名”罢了，她只希望找到灵丹治好永清的。啊，还有一个隐藏设定，不知道你有没有看过《罗小黑战记》，我记得里面有个细节就是阿根拥有两种能力的。所以我原本想好的是永清原本能够御水，但是一直隐藏了自己御冰的能力。又当然啦，想得多，写的少，也是因为我写不出来。

故事开头是，大概是我的一个梦，有天下午我躺在床上睡觉，我以为我醒了想要坐起来却发现身体动不了，可是我很确定自己很清楚地看见阿木狗就躺在不远处的狗窝上睡觉，于是那一刻我开始分不清自己究竟是醒着还是继续在梦中，一般情况下我的梦都是没有什么颜色，可是那天我很清楚地看到仍然是下午，仍然是天光，就是一模一样的现实，于是我连着喊了很多声阿木阿木，希望叫醒阿木狗，让它过来（PS也许是想让它喊醒我），可是阿木狗竟然一动不动，也不知道挣扎了多久，我终于明白自己仍然身在梦中。一旦我想明白了这点，现实和梦境的界限就能清晰地区分开来了，然后就自然地慢慢醒了。说来也怪，当我在梦里挣扎的时候越是用力挣扎反而越是醒不过来，可是当我放松了，知道自己总会醒过来的，不挣扎反而能够醒来。

第三个故事也是我的一个梦，原先的那个梦站在高处的是我，站在下面的是我的母亲，但是处境比小说里更加凶险，梦里一样是我的母亲让我听她的走最近的一条路，那是我觉得那条路走不通不肯走。这当然是对应着现实中的痛苦的，我的母亲这几年开始催我结婚，当然重点不是让我感受到幸福，只是想让我快点找个父母双全、身体健康的男人生孩子罢了。我跟她沟通过许多次，告诉她我是一个有自己的想法的人，不要干涉我，要尊重我，不过她做不到，她有一种在我看来不可理喻的信念，觉得她都是为了我好那么就不算是干涉和控制，而且每次我说她对我造成的伤害时，她都觉得自己没说什么。以前我非常非常迷信“非暴力沟通”这种东西，直到我被逼得发疯，不知怎地一口气把传统孝道观念带来的思想束缚也冲破了。令我感到非常无可奈何的一点是，当我平静温和地说出我的诉求时，母亲总是敷衍我承诺我不再提，非得我发疯骂她一句或者干脆再也不主动联系她，她居然才意识到我怎么好像生气了似的。唉，一说起我母亲的坏话，我真是滔滔江水绵绵不绝三天三夜都说不完呐。好啦，扯回来，所以说小说里并不是几分和世界格格不入的痛苦。不过这种痛苦以前确实是存在过的，许多年的时间里我都觉得自己是一个和别人不同的异类，总是听不懂别人说什么，或者别人不懂我说什么，后来……后来我发觉世人确实是很难百分百互相理解的，可是理解一部分是容易的，就好像我觉得我俩一样会在某些事上有共鸣，在某些特质上是相似的，我俩都是路痴，上大学的时候都偏好奇幻类的小说，都喜欢小狗，都很温和善良。

对了，我今天去爬山了，手脚并用的那种爬山，由于走的是驴友才走的野路，下山的时候路上就堆积了许多枯黄的落叶，踩落叶的声音真得好有意思，酥脆酥脆的。现在武汉城市街道里的落叶还比较少，就算有也早都被扫走了。

又及，在这里耕一则狗故事吧。今天下山以后遇到了一只狗，我见它很可爱就想摸，不过它戒备心很重，稍微靠近就会立马跳开，于是我蹲下伸手耐心等它靠近，它也只让摸了一下头就又迫不及待逃跑。然后我就和同伴走了，走了几步同伴提醒我它跟着我们，那时我想着背包里的食物可能都不太适合给狗吃，就容它跟着却没理它。再后来我们坐着休息，我翻书包才想到带的面包可以给狗吃，话说我那同伴是个执行力强的人，当时狗已经走了，她却说可以陪我再去找一找狗，我们回去刚下山的路口找了一下，狗不在，就还是走了。

我是那种心事不会压心里会在嘴上唠叨不停的人，后面又抱怨了一次自己要是当初想到把面包给狗吃就好了。结果呀，走了一小段路后同伴忽然指着下面说，哎，你想找的狗在那呢。还真是，我们当时是在水库上面的公路走，下面是枯水期水库露出土地的地方，狗果然就在下面溜达。我不知道狗的名字，就朝着狗的方位大喊“狗”，又想着人类通常看到狗会喊“狗勾”，也许这样喊的话狗知道是在喊它呢，就这么喊了好几声以后，我感觉到那狗隔着很遥远的距离和我对视了。可是我却看到狗往我的方向走了几步只是停下喝水，于是再一次放弃。可又是我那位同伴提醒我，她说，狗又朝我们过来了。这，还真是，水库边的大堤有台阶可以下去，我就翻了过去，拿出面包当着狗的面吃好几大口（以示无毒），由于我拿着面包递给狗它不愿意过来，最后只好放下面包，等我走远它才过去叼走面包，还跑更远才放心吃起来。

再后来，那狗又送了我们一程，走下大堤的时候，它还和我并排走着。
![dog-and-me](https://github.com/user-attachments/assets/f9666ac7-a749-4478-a3b9-38d58ecb7557)

**Reply by HushWay** [reply]
你解释的很清楚，不过在写法上我有一些想法：

1. 这三个故事是比较独立的，或许分成3篇或更多来写更好，不会这么累，每篇也更丰富一些；
2. 阿木的描写很生动，很可爱，但是其他人物刻画的笔墨比较少，或许多介绍一下人物的背景，或者通过细节突出一下人物的性格，可以拉近她们与读者的距离，更好地理解他们的行为。

你的小说还是很有风格的，情节简明，背后却隐含很多思考，而且会有很多意识流的东西，《信徒》那篇我就见识了。

> 可是儒家的那些所谓“名正言顺”的礼教束缚，在我看来非常虚伪，很多人都会下意识的以为有“名”的多少有点“实”，然而儒教统治了多久，名与实的分离就诞生了同样之久。

草台班子确实很多，不过关于“名正言顺”这几个字我有不同的理解。分享个故事，村子里面有个小学，收幼儿园到二年级的学生，我一年级在那读书，升二年级之前，听说这个小学要废除了，我们只好纷纷转学到别的学校。开学了，小学老师跑去开张收学费，结果没有收到一个学生，办不下去了，这个小学就真的废弃了。我想说的是，儒家确实存在礼教束缚，可名正言顺在我看来只是叙述了一种规律。写完突然觉得我们说的好像不是一个事情，算了，不改了。

> 非得我发疯骂她一句或者干脆再也不主动联系她，她居然才意识到我怎么好像生气了似的。

我也有过类似烦恼，好像语言无法起到和母亲沟通的作用。每次放假开学，她都要煮好多鸡蛋让我带着，可我讨厌吃水煮蛋。我和她说我不喜欢，她回：幸好你不喜欢，喜欢吃哪有那么多给你哦。唉。在你包里塞各种东西，甚至有次把拉链绷坏，和她说不需要，也不想带那么多，拉扯好久，终于商量好，不带那些乱七八糟的东西，结果火车上打开包一看，那些东西（包括水煮蛋）不知什么时候已经偷偷塞进来了。哭笑不得。有段时间她不厌其烦地劝我相亲，我没兴趣，她在我完全不知情的情况下，安排我和一个女生在街上见面。很难受，敷衍几句就告别了，然后对她发了好大一通脾气，可她那为我好的说辞只让我更生气。她为我好我知道，我都知道，可她的方式像是泥沼，让人接受不了。唉，其实作为儿子，想到这些场景，也会后悔，总后悔把脾气最差、最没有耐心的一面留给了她。其实我心情不好的时候很想她，想给她打电话，很不想看到她头上的白发。

> 再后来，那狗又送了我们一程，走下大堤的时候，它还和我并排走着。

哇哦，好温暖的画面呐。“拿出面包当着狗的面吃好几大口（以示无毒）”，这样以失示无毒狗狗是真的get到么？如果我带的是狗粮，是不是也得自己先尝一口，hhh

**Reply by earfanfan** [reply]
本来我是想当着狗的面拆开面包的包装来以示无毒的，然而之前以为找不到狗了，又懒得重新塞回包里，就干脆打开自己在吃。我那位同伴也是吐槽，就算我想吃几口意思意思也不用真吃好几大口，她说让我放嘴边假装吃几口就行。不过我当时想的是，怎么我猛吃几口然后再递给狗可是狗还不过来。下回我就知道应该少吃几口，放下食物，本人类走开，狗才放心。

关于小说的建议，我记住了，正好我打算把去年写了一半的残篇捡起来，预计下个月还会产出一篇小说，到时候写好了就喊你来看哈。

对了，你之前和益辉有过交流么，或者你也写博客么，我今天向他提起你爱看他的博客，他劝你放下遗憾，见[这里](https://github.com/earfanfan/yuanfan/discussions/461#discussioncomment-11070829)最后一楼。

你和你母亲的故事让我想起另一对母子的故事。从前有一个人类，他是家中长子，在他们那个年代读书是很辛苦的，当然每一个人年少的时候总会做些荒唐事，他上初中的时候和别人打牌把报名的学费输了，难受之余回家不知怎么跟母亲说，便蒙在被子里伤心难过还放话说不读书了，他的母亲不厌其烦地问他伤心的原因，终于他说了实话，本以为母亲会责骂他，哪知母亲竟然没有半句怨言，又重新卖了谷子凑了新的学费，此后他一路升学，读完了研究生就去了北京一所大学当老师。后来他结婚了有了孩子，不过妻子非常强势，许多年的时间里很少回家过年，直到近几年双亲年近八十，他才开始每年都回家过年。

不过他每次回家过年总是待不了几天就想早点离开，原因是受不了母亲一直不停地唠叨。吃饭的时候，母亲会一直念叨让他多吃饭，或者不停地把不同的菜碗端到他面前给他夹，每每这时年近六十的他也会发脾气，因为他有糖尿病吃多了饭等于吃毒药，给母亲解释过许多次母亲都不听。类似的事情不断地发生在他回家的时间里，每次他忍不住发脾气，母亲都是笑笑就过去，一样要坚持做自己想做的事。这个人就是我的大伯，有一次吃饭的时候我见大伯马上又要忍不住发脾气了，所以提前跟奶奶说让奶奶别操心了，可能我的语气也重了点，这时大伯又反过来替奶奶解释。他说，这已经是她能力范围内可以拿出来给我们的最好的东西了。

确实是这样的，城市里物资丰裕，什么都不缺。我奶奶一生都生活在村子里，她可以给我们最好的东西就是她做的饭菜了，不过她这几年真是年纪大了身体不太好，加上今年五一摔伤了肩膀更加没法做饭，所以大伯今年暑假回去几天，即便是我妈妈做的饭，我奶奶也很想我大伯能多吃几口。记得有一年国庆放假我回去，那几天阴雨连绵，奶奶风湿痛没法好好给我做饭，她在我临走前一夜气自己无能气得睡不着，还是第二天一早给我大姑姑打了好几个电话催她来给我做了些蒸菜，她才终于感到满足。

据说和父母沟通是会分好几个阶段的。第一个阶段是小时候，那时候我们不得自主，什么都要依赖家人，自然什么都得听他们的。第二个阶段就是现在，时代发展太快，父母辈爷爷奶奶辈跟不上时代，以前口口相传就能打听到可靠消息，现在打开手机一看各种虚假广告、意识形态、情绪焦虑，他们已经不知道该信哪个不该信哪个，可没有改变的是仍然拿我们当孩子看，学不会尊重我们，学不会放手，太多的差异导致我们和母亲沟通起来困难重重。不过我和你不同的是，我从小和爷爷奶奶生活，与爷爷奶奶的感情更深厚，前几年我也心疼母亲辛苦，会在她生日给她买蛋糕写上她的名字、给她唱生日歌、母亲节给她送花、平日里每周问候她给她买水果，听她讲一讲生活中的小事，放假回去帮她带孩子，不过今年所有耐心都被消耗殆尽，自从我发过疯以后我对她的心态完全改变了，我一点也不想她，甚至不会想起她。讲真，被逼得大发脾气让我的心灵也感到很受伤，只有远离她我才能获得平静，不然我很容易就会再次失控。传说中的第三个阶段，就是子女和父母共同努力，学会用互相尊重对方的心态去对待，建立一种相对平等的关系。不过我觉得我母亲应该是做不到的，当然可能是我现在对此感到很悲观，我的理由有这几点：其一，在母亲的思想观念中，父母和子女是有一种伦理秩序的，她认为天底下只有父母对子女最好，所以孩子应该接受父母的好意而不是自己做出改变；其二，母亲生活在大家庭中，一生不曾独立生存过，而且她仍然用她的人生经验去看待我的人生，所以无法理解拥有了独立自我的孩子，还以为我也要完成撒人生任务，也不明白我为撒不快点找个男人生孩子然后为孩子奉献一生。唉，这个时代不仅对于我们来说充满了挑战，对她们来说也是越来越多考验，现在考验她们的就是，不要把自己的意志强加在孩子身上。

说实话，从现在的我的视角看，我的母亲只是年龄增长，但她很多时候都像个小孩子一样，让我感到无语、无奈、无能为力。其实我也在想，如果我这一生真得没生个孩子出来，会不会某种程度上是剥夺了她的幸福、破坏了她对美好人生的期盼。不过，即便这样，真得不是我的错，也不是她的错，是时代变化太快，所以我的思想和她如此不同。

你关于“名正言顺”的不同理解，我看明白了。很多事物都值得辨证去看待的。如果某一天我们变得非常固执地只愿意看到事物的一个方面，那一定是我们那时太需要只相信那一面了。说来也怪，我俩交流大多在说些感受和事情，思考占的比例很小，明明我俩应该都算是会读书且善于思考的人。以后有机会的话，也许咱们也可以多交流费脑子的主题。不过我明天要出差，得去收拾行李了，我去也，拜拜，祝你一切顺利吧。

**Reply by HushWay** [reply]
谢谢你给我讲那么多，让我有新的视角去理解和父母的沟通。祝你平安，一切顺利，拜拜。（PS：我也想写点东西了，还没想好，到时候可能和你分享）

**Reply by earfanfan** [reply]
好啊，我的邮箱是 Fanyuan@protonmail.com，回头你愿意的话，也可以给我写信。
